src/TodosFilter.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+export const TodosFilter = ({ selectedFilter, setFilter }) => {
+  const handleFilter = (event) => {
+    event.preventDefault();
+
+    setFilter(event.target.name);
+  };
+
+  return (
+    <ul className="filters">
+      <li>
+        <a
+          href="#/"
+          className={classnames({ selected: selectedFilter === 'All' })}
```

Comment:
```suggestion
          className={classnames({ selected: selectedFilter === FILTERS.All})}
```
import FILTERS object to this file, and use it instead of this string